cypress/e2e/universal-search.test.ts

```diff
@@ -35,6 +39,36 @@ describe('Universal search bar', () => {
     cy.contains('UNI is the governance token for Uniswap').should('exist')
   })
 
+  it('should go to the right result when recent results are shown and you search something new', () => {
+    // Search for uni token by name.
+    cy.get('[data-cy="search-bar-input"]').last().clear().type('uni')
+    cy.get('[data-cy="searchbar-token-row-UNI"]')
+      .should('contain.text', 'Uniswap')
+      .and('contain.text', 'UNI')
+      .and('contain.text', '$')
+      .and('contain.text', '%')
```

Comment:
Verifying the row exists should be sufficient (here and with ETH).

```suggestion
    cy.get('[data-cy="searchbar-token-row-UNI"]')
```